.gitignore

```diff
@@ -9,7 +9,8 @@ tests/temp
 
 #binaries
 bin/phpunit
-bin/phpcs
+bin/sabre-cs-fixer
```

Comment:
@staabm we have our own 'binary' that's just `php-cs-fixer` that automatically loads our configuration. However, we can't prevent composer from also creating `php-cs-fixer` in the `bin/` directory, even if we don't need it.